### PR TITLE
add page CF Philosophy

### PR DESCRIPTION
**Pull request by Hamza-Luciano**

### CODE REVIEW

**Review by FrancoisSalmon — DISMISSED** [review]
ok

File: frontend/src/components/TextPhilosophy/index.jsx [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+                  img={`./src/assets/images/cards/players/${
+                    player.img || "test.png"
+                  } 
+                `}
+                  number={player.number}
+                />
+              );
+            })}
+        </div>
+        <div className="descriptifText">
```
Comment (LuigiMenez) [inline frontend/src/components/TextPhilosophy/index.jsx:81]: In english please "descriptiveText"

File: frontend/src/components/TextPhilosophy/index.jsx [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+                  img={`./src/assets/images/cards/players/${
+                    player.img || "test.png"
+                  } 
+             `}
+                  number={player.number}
+                />
+              );
+            })}
+        </div>
+        <div className="descriptifText">
```
Comment (LuigiMenez) [inline frontend/src/components/TextPhilosophy/index.jsx:110]: same

File: frontend/src/components/TextPhilosophy/index.jsx [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+                  img={`./src/assets/images/cards/players/${
+                    player.img || "test.png"
+                  } 
+                `}
+                  number={player.number}
+                />
+              );
+            })}
+        </div>
+        <div className="descriptifText">
```
Comment (LuigiMenez) [inline frontend/src/components/TextPhilosophy/index.jsx:140]: Same

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Fix minor

**Review by FrancoisSalmon — APPROVED** [review]
ok for me

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**